src/Umbraco.Core/Services/Implement/RelationService.cs

```diff
@@ -294,6 +294,17 @@ public IEnumerable<IUmbracoEntity> GetPagedParentEntitiesByChildId(int id, long
             }
         }
 
+        public IEnumerable<IUmbracoEntity> GetPagedParentEntitiesByChildIds(int[] ids, long pageIndex, int pageSize, out long totalChildren,
```

Comment:
Can we have a unit test for this one? Should be pretty straight forward to use the other RelationServiceTests methods as a starting point.

Reply:
I added a unit test for this one. I noticed a lot more methods on this Service are untested. But that is maybe something for another PR